src/core/database/results/parse-base-node.ts

```diff
@@ -4,9 +4,10 @@ import { DateTime } from 'luxon';
 export interface BaseNode {
   id: string;
   createdAt: DateTime;
+  sortValue?: string;
 }
 
 export const parseBaseNodeProperties = (node: Node<BaseNode>) => {
-  const { id, createdAt } = node.properties;
-  return { id, createdAt };
+  const { id, createdAt, sortValue } = node.properties;
+  return { id, createdAt, sortValue };
```

Comment:
We shouldn't ever need to pull the sortValue out of the database. It will only be used in queries.